src/test/java/teammates/test/cases/datatransfer/FeedbackSessionAttributesTest.java

```diff
@@ -138,4 +138,28 @@ public void testValidate() {
         return Arrays.asList(feedbackSessionNameError, courseIdError, creatorEmailError);
     }
 
+    @Test
+    public void testGracePeriodValidate() {
```

Comment:
Combine this with `testValidate` instead. Simply modify the builder and add the new error to `buildExpectedErrorMessages`.